zebrad/src/application.rs

```diff
@@ -37,6 +43,98 @@ pub fn app_config() -> config::Reader<ZebradApp> {
     config::Reader::new(&APPLICATION)
 }
 
+/// Toplevel entrypoint command.
+///
+/// Handles obtaining toplevel help as well as verbosity settings.
+#[derive(Debug, Options)]
+pub struct EntryPoint {
```

Comment:
Can you put this struct in a submodule of `application`?
Can you link to the `abscissa` code that you copied?

The documentation will help us when we upgrade abscissa to the latest version.
(Linking to copied code is required for licensing reasons. Having a separate module is useful but optional.)